test-cases/performance/perf-regression-latency-mv-read-concurrency.yaml

```diff
@@ -0,0 +1,31 @@
+test_duration: 680
+prepare_write_cmd: ["cassandra-stress write no-warmup cl=ALL n=100000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=100 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..2000000",
+                    "scylla-bench -workload=sequential -mode=write -replication-factor=2 -partition-count=10 -partition-offset=0 -clustering-row-count=1000000 -clustering-row-size=uniform:100..5120 -concurrency=200 -rows-per-request=10 -timeout=30s -connection-count 200  -consistency-level=all",
```

Comment:
looks like 'large partition' workload, is this 'special'?

Reply:
it is just prepare separate table with 'large partition"

 'special workload", it is a combination of table with large partitions + MV + small write in table that causes a lot of MV updates 
 stress_cmd_no_mv and stress_cmd_mv are the same but difference in  latency during executing this command is huge https://github.com/scylladb/qa-tasks/issues/1706#issuecomment-2351624595